app/image/image-details-directive.html

```diff
@@ -93,5 +98,30 @@
       </a>
     </div>
   </tab>
+  <tab>
+    <tab-heading>
+      Labels
+    </tab-heading>
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Key</th>
+            <th>Value</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr ng-repeat="(key, value) in imageDetails.labels">
+            <td>{{key}}</td>
+            <td>{{value}}</td>
+          </tr>
+        </tbody>
+      </table>
+    <div class="list-group">
+      <a ng-repeat="img in imageAncestry" href="image/{{img}}" class="list-group-item" ng-class="{active: imageDetails.id==img}">
```

Comment:
I didn't see this ancestry list in your screenshots. Is this a left-over of some sort?